homeassistant/components/image_processing/gocr.py

```diff
@@ -0,0 +1,150 @@
+"""
+Local optical character recognition processing using GOCR.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/image_processing.gocr/
+"""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.core import split_entity_id
+from homeassistant.components.image_processing import (
+    PLATFORM_SCHEMA, ImageProcessingEntity, CONF_SOURCE, CONF_ENTITY_ID,
+    CONF_NAME)
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_UNRECOGNIZED = 'unrecognized'
+CONF_CHARS = 'chars'
+CONF_EXTRA_ARGUMENTS = 'extra_arguments'
+CONF_HEIGHT = 'height'
+CONF_ROTATE = 'rotate'
+CONF_GOCR_BIN = 'gocr_bin'
+CONF_THRESHOLD = 'threshold'
+CONF_WIDTH = 'width'
+CONF_X_POS = 'x_position'
+CONF_Y_POS = 'y_position'
+CONF_NEGATE = 'negate'
+
+DEFAULT_BINARY = 'gocr'
+DEFAULT_UNRECOGNIZED = '_'
```

Comment:
What does this default value do?

Reply:
It is the default string GOCR use to replace any character that are unable to be recognized above the confidence threshold.

Reply:
I removed it in the end.